src/plan/nogc/mutator.rs

```diff
@@ -1,18 +1,36 @@
 use crate::plan::barriers::NoBarrier;
 use crate::plan::mutator_context::Mutator;
 use crate::plan::mutator_context::MutatorConfig;
+use crate::plan::mutator_context::{
+    create_allocator_mapping, create_space_mapping, ReservedAllocators,
+};
 use crate::plan::nogc::NoGC;
-use crate::plan::AllocationSemantics as AllocationType;
+use crate::plan::AllocationSemantics;
 use crate::plan::Plan;
 use crate::util::alloc::allocators::{AllocatorSelector, Allocators};
 use crate::util::{VMMutatorThread, VMWorkerThread};
 use crate::vm::VMBinding;
-use enum_map::enum_map;
-use enum_map::EnumMap;
+use enum_map::{enum_map, EnumMap};
+
+const RESERVED_ALLOCATORS: ReservedAllocators = ReservedAllocators {
+    n_bump_pointer: 3,
```

Comment:
Without `nogc_multi_space`, this should be `1`.

Reply:
This is not used if `nogc_multi_space` is disabled. I've renamed it to `MULTI_SPACE_RESERVED_ALLOCATORS` and wrote some comments to make it more clear.

Reply:
I see. What if we just remove the feature and use `nogc_multi_space` as the default? Is there any reason we still want to keep the current (mapping all allocation semantics to bump pointer #0)?